pkg/kubectl/cmd/annotate.go

```diff
@@ -36,25 +36,26 @@ import (
 
 // AnnotateOptions have the data required to perform the annotate operation
 type AnnotateOptions struct {
+	// Filename options
 	resource.FilenameOptions
```

Comment:
I'd like to avoid anonymous inclusion where possible.  The method aliasing seems to encourage bad habits (see `Factory`) and gain for non-serialized types isn't very significant.  I won't block on it.